keymap.cson

```diff
@@ -26,6 +26,8 @@
 
 # window/pane navigation
 'body, atom-text-editor':
+  'shift-cmd-y': 'pane:show-next-item'
+  'shift-cmd-h': 'pane:show-previous-item'
```

Comment:
I think this is fine, but I use `cmd-shift-[`and `cmd-shift-]` for this, since that's standard across the OS (even though atom has it additionally mapped `cmd-alt-left/right`.